app/jenkins-for-jira-ui/src/common/util/jenkinsConnectionsUtils.ts

```diff
@@ -12,19 +12,26 @@ export const setName = async (event: React.ChangeEvent<HTMLInputElement>, setSer
 	setServerName(event.target.value);
 };
 
+export const sanitizeInput = (input: string): string => {
```

Comment:
Realised I neglected to sanitise the input so accounting for that now.

Reply:
if you wanted a more generic handler theres this [fella](https://www.npmjs.com/package/he) looks to encode most things... never tried it tho